src/backend/distributed/commands/common.c

```diff
@@ -111,8 +113,10 @@ PreprocessAlterDistributedObjectStmt(Node *stmt, const char *queryString,
 	const DistributeObjectOps *ops = GetDistributeObjectOps(stmt);
 	Assert(ops != NULL);
 
-	ObjectAddress address = GetObjectAddressFromParseTree(stmt, false);
-	if (!ShouldPropagateObject(&address))
+	List *addresses = GetObjectAddressFromParseTree(stmt, false);
+	Assert(list_length(addresses) == 1);
+
+	if (!ShouldPropagateObjects(addresses))
```

Comment:
I think `ShouldPropagateAnyObject` is easier to understand what the function does